.ldrelease/publish-circleci.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Run this in publish step after all version information have been updated.
+set -ev
+
+brew install circleci
+
+# Read 2 arguments from the command line so we can debug this script.
+# Argument 1 is the release version. Defaults to releaser env variable.
+RELEASE_VERSION=${1:-$LD_RELEASE_VERSION}
+
+# Argument 2 is the circleci token. Defaults releaser circleci secret.
+CIRCLECI_CLI_TOKEN=${2:-"${LD_RELEASE_SECRETS_DIR}/circleci_token"}
+CIRCLECI_CLI_HOST="https://circleci.com"
+
+# Validate circleci orb config. No publishing done on this step.
+circleci orb validate build/package/circleci/orb.yml || (echo "Unable to validate orb"; exit 1)
```

Comment:
We have a `make` version of this do we want to call that so if there's changes required we only need to do them in one spot.

Reply:
I think we should remove those circleci make targets and use this shell script to publish to circle. I've added a comment to do this when we have a github owner token but will leave them for now so we can continue publishing the old way. In the future, all publishing to circleci orbs should be done through `publish-circleci.sh`.